// grunt/Gruntfile.js
'use strict';

module.exports = function(grunt) {

    // Project configuration.
    grunt.initConfig({
        pkg: grunt.file.readJSON('package.json'),
        cssmin: {
            options: {
                shorthandCompacting: false,
                roundingPrecision: -1
            },
            target: {
                files: [{
                    expand: true,
                    cwd: 'release/css',
                    src: ['*.css', '!*.min.css'],
                    dest: 'release/css',
                    ext: '.min.css'
                }]
            }
        }
    });

//     watch: {
//         gruntfile: {
//             files: '<%= jshint.gruntfile.src %>',
//             tasks: ['jshint:gruntfile']
//         },
//         lib: {
//             files: '<%= jshint.lib.src %>',
//             tasks: ['jshint:lib', 'nodeunit']
//         },
//         test: {
//             files: '<%= jshint.test.src %>',
//             tasks: ['jshint:test', 'nodeunit']
//         },
//     },
// });

    // These plugins provide necessary tasks.
    grunt.loadNpmTasks('grunt-contrib-cssmin');
    
    // Default task.
    grunt.registerTask('default', ['cssmin']);

};
